frontend/public/components/utils/headings.tsx

```diff
@@ -133,7 +133,11 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
           ) : (
             <div className="co-m-pane__name co-resource-item">
               {kind && <ResourceIcon kind={kind} className="co-m-resource-icon--lg" />}{' '}
-              <span data-test-id="resource-title" className="co-resource-item__resource-name">
+              <span
+                data-test-id="resource-title"
+                data-test="resource-title"
```

Comment:
Don't add both. Either update old tests for `data-test` or use the legacy `data-test-id`

Reply:
fixed, reverted back to just using `data-test-id`.